ant-protocol/src/messages/query.rs

```diff
@@ -84,6 +84,24 @@ pub enum Query {
         /// Address of the record.
         address: NetworkAddress,
     },
+    /// Get a Merkle payment candidate quote from a node
+    ///
+    /// Node signs its current state (metrics + reward address) with the
+    /// provided timestamp, creating a commitment that cryptographically
+    /// proves the binding between PeerId and RewardsAddress.
+    GetMerkleCandidateQuote {
```

Comment:
I'd suggest use the same Quote req/rsp.
It's not only make the merkle quotes to be outstanded from normal quotes, having less backward compatible issue or avoid being vlunerable to be identified out and gaming, but also bearing the same `storage challenge` feature which could be useful in future.